hackathon_site/event/static/event/js/application.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function() {
+    $('#id_q1, #id_q2, #id_q3').characterCounter();
```

Comment:
Just to be safe, you should use `{{ form.q1.id_for_label }}` etc here instead of hard-coding the ids.

`$('#{{ form.q1.id_for_label }}, #{{ form.q2.id_for_label }}, #{{ form.q3.id_for_label }}').characterCounter();`